DualCameraDemo/DualCameraDemo/DualCameraDemoApp.swift

```diff
@@ -1,30 +1,84 @@
 import DualCameraKit
+import Observation
 import SwiftUI
 
-@main
+@main 
 struct DualCameraDemoApp: App {
+    @State private var controllerManager = ManagedDualCameraController()
```

Comment:
i don't love this pattern. consider if an alternative works here. maybe just a simple reducer